### ISSUE TITLE
Unable to benchmark since v33 - version prior to v33 are ok.

### ISSUE THREAD

**Issue by RckMan** [angelbbs/NiceHashMinerLegacy#101]
Unable to re-benchmark since v33. All attempts to benchmark give the following dialog:

"Current benchmark settings are already executed. There is nothing to do."

I re-installed and had the installer do a clean wiping all settings. Same thing - unable to benchmark.  I can delete all of the benchmark .json files and still unable to benchmark.

**Comment by angelbbs** [comment]
click right mouse button on selected algo

**Comment by RckMan** [comment]
Yes, even trying to benchmark a single algo gives the same error message.  Only tried on Win10 2004 build.

**Comment by RckMan** [comment]
Here is a video showing the error with v33.1
https://youtu.be/vsgLUrqQCGQ

**Comment by angelbbs** [comment]
Thanks.  Will be fix in next release

**Comment by angelbbs** [comment]
I can't repeat it. Please send a video with problem starting with program run

**Comment by sizhuk** [comment]
I have tha same error after upgrading to v34. Win10 LTSB. NHML is able to benchmark AMD cards and I can see in the config folder files "benchmark ... 1002 " that is for AMD and no one typical benchmark file for nVidia

**Comment by angelbbs** [comment]
Same errorr? )) Where log.txt?

**Comment by RckMan** [comment]
FYI, I just tried 34.1 and I am once again able to benchmark - it is working perfectly. Thanks! Go ahead and close this issue.

**Comment by sizhuk** [comment]
))
Same error on my friend's PC. She has 2 1060 and automatically updated 34.1. There was not any problems until she installed the latest drivers. After this NHML detects card, but forgets about previously saved benchmark results)) Of course no logs, but a video)) Right click does not work. In this case it was able to benchmark the 2nd card and manually copy/paste results to the 1st in settings window

https://user-images.githubusercontent.com/22453838/103311446-0ef24000-4a23-11eb-9696-c1447fb3ef94.mp4

**Comment by angelbbs** [comment]
Send logs\log.txt

**Comment by sizhuk** [comment]
[log.txt](https://github.com/angelbbs/NiceHashMinerLegacy/files/5754001/log.txt)

**Comment by angelbbs** [comment]
Delete C:\Program Files\NVIDIA Corporation\NVSMI folder & delete C:\NHM\nvml.dll file then reinstall nvidia drivers

**Comment by sizhuk** [comment]
Sorry for that latency. Only today the PC with these miner became online after a holidays. I used your recommendations and it works. Thank you.
